packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -198,13 +200,15 @@ def late_arriving_tx_hashes(self) -> List[str]:
         return [h for hashes in parsed_hashes for h in hashes]
 
     @property
-    def most_voted_check_result(self) -> str:
+    def most_voted_check_result(self) -> str:  # pragma: no cover
```

Comment:
If both are not used we should probably remove the properties. Please check with @Adamantios

Reply:
This property is only used for type checking, so it is safe to not cover it:

https://github.com/valory-xyz/open-autonomy/blob/79f617df66d2d19cffba43e66768828f6c7702e2/packages/valory/skills/transaction_settlement_abci/rounds.py#L428